packages/es-components/rollup.config.js

```diff
@@ -0,0 +1,14 @@
+import babel from 'rollup-plugin-babel';
+
+export default {
+  input: 'src/index.js',
+  output: {
+    file: 'lib/index.js',
+    format: 'esm'
+  },
+  external: [
```

Comment:
do we need to add any externals? react or react-dom? or does that not matter because they're peerDependencies?

Reply:
Without the `node-resolve` plugin rollup doesn't know to look in `node_modules` and treats any such module it can't find as external. Adding those modules to this `external` array suppresses any console warnings during the build.

Reply:
right, so I guess does it make sense to call those out here? or are we ok throwing our hands up and ignoring those console warnings?

Reply:
I'm leaning towards the latter, personally. For most contributors those warnings are not going to be seen. The biggest thing though is how volatile our dependency list can be. We add/remove dependencies fairly regularly, so this `array` should reflect that. I don't think it is feasible or worth the effort to automate such checks that items needs to be added/removed. If we want to suppress those warnings I could, but I don't think people are running the build script locally enough to complain about the noise.